test/image_test.py

```diff
@@ -255,6 +256,26 @@ def test_save(self):
                 # clean up the temp file, comment out to leave tmp file after run.
                 os.remove(temp_filename)
 
+    def test_save_to_fileobject(self):
+        s = pygame.Surface((1, 1))
+        s.fill((23, 23, 23))
+        bytes_stream = io.BytesIO()
+
+        pygame.image.save(s, bytes_stream)
+        bytes_stream.seek(0)
+        s2 = pygame.image.load(bytes_stream, "tga")
+        self.assertEqual(s.get_at((0, 0)), s2.get_at((0, 0)))
+
+    def test_save_tga(self):
+        s = pygame.Surface((1, 1))
+        s.fill((23, 23, 23))
+        with tempfile.NamedTemporaryFile(suffix=".tga") as f:
```

Comment:
It looks like this is failing on windows. This is because on windows, you can't open the file if it's open again.

```python
with tempfile.NamedTemporaryFile(delete=False) as afile:
    temp_filename = afile.name

pygame.image.save(s, temp_filename)
s2 = pygame.image.load(temp_filename)
self.assertEqual(s2.get_at((0, 0)), s.get_at((0, 0)))
```

Reply:
You can tell I'm not coding on a Windows machine. Thanks to CI for this problem!

I've amended the code trying to make sure the temp file gets deleted even if the test fails.

Reply:
Nice one! Looks like it's passing on windows now.